hooks/zircon.sh

```
@@ -8,21 +8,34 @@ set -e
+mkdir -p "$ZIRCON_LIBZR_DIR/lib"
+mkdir -p "$ZIRCON_LIBZR_DIR/include"
 
 cargo build --release
 
 cp target/release/zrc "$ZIRCON_BIN_DIR/"
 cp target/release/zircop "$ZIRCON_BIN_DIR/"
-
 cp -r include/* "$ZIRCON_INCLUDE_DIR/"
 
+# build std using the fresh compiler
+make -C libzr all-opt ZRC=$ZIRCON_BIN_DIR/zrc
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)